arviz/plots/dotplot.py

```diff
@@ -0,0 +1,142 @@
+import math
+import numpy as np
+
+from ..rcparams import rcParams
+from .plot_utils import get_plotting_function
+
+
+def plot_dots(
+    values=None,
+    binwidth=None,
+    dotsize=1,
+    stackratio=1,
+    hdi_prob=None,
+    rotated=False,
+    dotcolor="grey",
+    intervalcolor="red",
+    markersize=None,
+    figsize=None,
+    linewidth=None,
+    point_estimate="auto",
+    quantiles=None,
+    point_interval=None,
+    plot_kwargs=None,
+    interval_kwargs=None,
+    backend=None,
+    backend_kwargs=None,
+    **kwargs
+):
+
+    """Plot distribution as dot plot.
```

Comment:
Plot distribution as quantile dot plot

Reply:
"... as a dot plot or quantile dot plot" I think would be best, plus as you commented above, references to both dotplot and quantile dotplot explanations.

Reply:
I also think "Plot distribution as dot plot or quantile dot plot" sounds better